config/grunt/task-config/shell.js

```diff
@@ -209,6 +209,12 @@ module.exports = function( grunt ) {
 				callback: throwUncommittedChangesError,
 			},
 		},
+		"readme-reset-txt": {
```

Comment:
```suggestion
		"reset-readme-txt": {
```

Reply:
tis was to keep in in line with this function: "composer-reset-lock": {
			command: "git checkout composer.lock",
			options: {
				failOnError: false,

			},
		},

as this does the same for the composer.lock file.